packages/react-router/__tests__/Routes-test.tsx

```diff
@@ -1,22 +1,37 @@
 import * as React from "react";
-import { create as createTestRenderer } from "react-test-renderer";
+import * as ReactDOM from "react-dom";
+import { act } from "react-dom/test-utils";
 import { MemoryRouter as Router, Routes, Route } from "react-router";
 
 describe("A <Routes>", () => {
+  let node: HTMLDivElement;
```

Comment:
Note: I updated all of the tests in this file to use inline snapshots so its more immediately clear what each one is testing for. Personally prefer our tests that query the DOM vs. a JSON blob, but otherwise they're essentially the same!

Reply:
Agree. I like inline snapshots way better than opening a separate file to check what the snapshot prints. Good call 👍